.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  make-test:
+    name: Unit tests
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 'stable'
```

Comment:
Does it make sense to instead use `go-version-file` here?
```
          go-version-file: 'go.mod'
```

Reply:
I believe the intent with fswalker is that it's compatible with effectively any go version in the 1.x branch, so latest  stable should be fine.